package.json

```diff
@@ -61,6 +61,7 @@
     "lodash.uniq": "~3.2.2",
     "lodash.without": "~3.2.1",
     "mkdirp": "~0.5.1",
+    "node-fs-extra": "~0.8.1",
```

Comment:
I think you meant to require `fs-extra` here and in the code as well - `node-fs-extra` points to an old fork for which the linked source repository doesn't exist anymore.